common/metrics/defs.go

```diff
@@ -837,6 +839,10 @@ const (
 	GetAvailableIsolationGroupsScope
 	// TaskValidatorScope is the metric for the taskvalidator's workflow check operation.
 	TaskValidatorScope
+
+	// FrontendGlobalRatelimiter is the metrics scope for frontend.GlobalRatelimiter
+	FrontendGlobalRatelimiter
+
```

Comment:
moved from frontend to shared collection, as otherwise it wasn't available in history